docs/integrations/android-specific-workflow-steps/detekt.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Detekt
+metaTitle: Detekt
+metaDescription: Detekt
+sidebar_position: 16
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Detekt
+
+[Detekt](https://detekt.dev) is a [Kotlin](https://kotlinlang.org) static code analysis tool used in Android development to identify issues, enforce standards, and enhance code quality. It checks for code smells, performance problems, bugs, and adherence to best practices, offering configurable rules and plugins for customization. Integration into development workflows ensures clean and efficient Kotlin codebases.
+
+The Appcircle Detekt step executes the Detekt Gradle task. For further details, please refer to the [Run detekt using the Detekt Gradle Plugin](https://detekt.dev/docs/gettingstarted/gradle/) documentation.
+
+### Prerequisites
+The workflow steps that need to be executed before running the **Detekt** workflow step, along with their respective reasons, are listed in the table below.
+
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [Git Clone](https://docs.appcircle.io/workflows/common-workflow-steps/git-clone) | The **Git Clone** step must be executed prior to the **Detekt** step to fetch the repository before performing code checks. |
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/android-workflow-components-detekt_1.png'/>
+
+### Input Variables
+For each component, specific input variables are required for its operation on your system. The input variables necessary for the **Detekt** are as follows:
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/android-workflow-components-detekt_2.png'/>
+
+| Variable Name                 | Description                                    | Status |
+|-------------------------------|------------------------------------------------|--------|
+| `$AC_DETEKT_TASK`             | Specify the **Detekt** task name. The default value is `detekt`. | Required |
+| `$AC_DETEKT_EXTRA_PARAMETERS` | Additional command-line parameters for **Detekt**. | Optional |
+| `$AC_DETEKT_SAVE_REPORT`      | Specify whether the **Detekt** report will be saved.  If set to `true`, report files will be saved into the artifacts folder. The default value is `false`. | Optional |
```

Comment:
@ftmzhrsckldr Detekt also has the  AC_DETEKT_OUTPUT_PATH input variable that was added in issue BE-1762. Below PR's changes are applied to the `testing` branch at `appcircle-workflow-components`.
- https://github.com/appcircleio/appcircle-workflow-components/pull/364/files

Currently, they're deployed to the `dev` environment, but they will be released soon.

**Note:** There is a line there that appends `gradlew` folder path to the user input.
```ruby
detekt_output_path = get_env_variable('AC_DETEKT_OUTPUT_PATH') || "#{ac_module}/build/reports"
# ...
if save_report == 'true'
# ...
  detekt_output_path = File.join(gradlew_folder_path, detekt_output_path)
# ...
end
# ...
```
[ref.](https://github.com/appcircleio/appcircle-detekt-component/blob/main/main.rb)

We should add information about this concatenation so that users can input it correctly.